src/main/java/springboot/Controller/ItemController.java

```diff
@@ -0,0 +1,16 @@
+package springboot.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import springboot.Model.*;
+
+@Controller
+public class ItemController {
+
+    @GetMapping(value = "/itemPage")
+    public String itemPage() {
```

Comment:
Should have something like model.addAttribute("items", service.getAllItems()) because as of now, it causes an error when it tries to go to itemPage.html as 'items' is null

Reply:
It is fixed now